sotodlib/site_pipeline/record_qa.py

```diff
@@ -98,9 +107,14 @@ def main(config):
                         f"Processing metric {m._influx_meas}.{m._influx_field} failed.",
                         exc_info=True
                     )
+                    n_fail += 1
```

Comment:
I think the idea would be that this would only count as a `n_fail += 1` (raised to prefect error level) if `visit_count > max_retry`

This is the right track -- thanks.

Reply:
right, that make sense ! thanks